packages/zmq-android.4.0-8/findlib

```diff
@@ -0,0 +1 @@
+zmq
```

Comment:
That doesn't make any sense, ocamlfind does not use these "findlib" files. Are you sure there was no other difference? Can you check one more time please?